src/assets/iconDiamondActive.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="17.498" height="14" viewBox="0 0 17.498 14">
+  <g id="Icon_Heart" data-name="Icon/Heart" transform="translate(-0.251 0)">
+    <path id="Shield" d="M8.75,14a.616.616,0,0,1-.439-.183L.185,5.667a.645.645,0,0,1-.11-.752L2.52.332A.622.622,0,0,1,3.069,0H14.431a.622.622,0,0,1,.549.332l2.444,4.583a.644.644,0,0,1-.111.752l-8.125,8.15A.616.616,0,0,1,8.75,14ZM3.6,1.5,1.7,5.06l7.052,7.072L15.8,5.06,13.9,1.5Zm4.824,9.8H8.4a.178.178,0,0,1-.155-.139l0-.056.425-3.412a.176.176,0,0,0-.124-.166l-.057-.009-3.048.011a.188.188,0,0,1-.1-.031A.173.173,0,0,1,5.265,7.3l.025-.05L9.2,1.874A.18.18,0,0,1,9.345,1.8l.027,0a.177.177,0,0,1,.15.143V2L9.005,5.36l0,.026a.176.176,0,0,0,.123.166l.058.008,2.906-.006a.184.184,0,0,1,.1.029.171.171,0,0,1,.075.191l-.023.051-3.67,5.4A.18.18,0,0,1,8.421,11.3Z" transform="translate(0.25)" fill="#67f7b8"/>
```

Comment:
@stanislav-slavov you may use `currentColor` instead of `#67f7b8` to be able to change this color from styles and avoid icon duplication

Reply:
I guess we have to use inline svg in order to apply currentColor. This may be a separate task, to optimize the resources.